data/records/delphi-documentation-logbooks.json

```diff
@@ -0,0 +1,233 @@
+[
+  {
+    "abstract": {
+      "description": "These are the DELPHI online logbooks. There used to be three people on shift during data taking, having different roles. Available logs books are these of the shift leader, the data aquisition logs, the slow control maestro and the quality checker. In addition, for later years, logbooks from the HPC detector shifter are preserved and available here."
+    },
+    "accelerator": "CERN-LEP",
+    "collaboration": {
+      "name": "DELPHI collaboration",
+      "recid": "80050"
+    },
+    "collections": [
+      "DELPHI-documentation-manuals"
+    ],
+    "date_created": [
+      "2000"
```

Comment:
We should probably include all the years in `date_created`, since this field corresponds to collision data taking. There can be multiple values, so we can list 1991, 1992, etc.  This will be good for search facets.